tests/raftstore_cases/test_prevote.rs

```diff
@@ -104,13 +106,15 @@ fn test_prevote<T: Simulator>(
 
     // Prepare to listen.
     let rx = attach_prevote_notifiers(cluster, detect_during_recovery.0);
```

Comment:
I think we don't need to catch prevote messages again after recovery. Or we can add `clear_send_filters_by<F: Fn(Filter) -> bool>(filter: F)` to `Cluster` to remove partition filters only. Considering the later solution is a little complex, I think just remove the detection is ok.

Reply:
I reflected this change, I think it's a good idea. :)